index.js

```diff
@@ -6,10 +6,12 @@ module.exports = Envie
 Envie.Envie = Envie
 Envie.Joi = Joi
 
-function Envie (description, values, options) {
-  if (!(this instanceof Envie)) return new Envie(description, values, options)
-  if (!values) values = process.env
-  if (!options) options = { noDefaults: false }
+function Envie (description, options = {}) {
+  if (!(this instanceof Envie)) return new Envie(description, options)
+  const {
+    values = process.env,
```

Comment:
It's a pretty normal way to deal with default values nowadays since destructuring is becoming the norm. This creates the variable `values` from `options.values` with the fallback of `process.env`.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values_2


You can see that the tests pass.

Reply:
I see, I mistook it for `values: process.env`